source/client-side-encryption/client-side-encryption.md

```diff
@@ -0,0 +1,2483 @@
+[libmongocrypt](#libmongocrypt). Drivers MUST include the following in the documentation for MongoClient:
+
+> Supplying an `encryptedFieldsMap` provides more security than relying on an `encryptedFields` obtained from the
+> server. It protects against a malicious server advertising a false `encryptedFields`.
+
+#### bypassQueryAnalysis
+
+See [Why is bypassQueryAnalysis needed?](#why-is-bypassqueryanalysis-needed).
+
+### Queryable Encryption Create and Drop Collection Helpers
+
+A collection supporting Queryable Encryption requires an index and three additional collections.
+
+<div id="GetEncryptedFields">
+
+#### Collection `encryptedFields` Lookup (GetEncryptedFields)
+
+The convenience methods support the following lookup process for finding the `encryptedFields` associated with a
+collection.
+
+Assume an exposition-only function `$GetEncryptedFields(opts, collName, dbName, askDb)$`, where `$opts$` is a set of
+options, `$collName$` is the name of the collection, `$dbName$` is the name of the database associated with that
+collection, and `$askDb$` is a boolean value. The resulting `encryptedFields` `$EF$` is found by:
+
+1. Let `$QualName$` be the string formed by joining`$dbName$` and `$collName$` with an ASCII dot `"."`.
+2. If `$opts$` contains an `"encryptedFields"` property, then `$EF$` is the value of that property.
+3. Otherwise, if `AutoEncryptionOptions.encryptedFieldsMap` contains an element named by `$QualName$`, then `$EF$` is
+   the value of that element.
+4. Otherwise, if `$askDb$` is `true`:
+   1. Issue a `listCollections` command against the database named by $dbName$, filtered by `{name: <collName>}`. Let
+      the result be the document `$L$`.
+   2. If `$L$` contains an `options` document element, and that element contains an `encryptedFields` document element,
+      `$EF$` is `$L$` `\["options"]["encryptedFields"]`.
+   3. Otherwise, $EF$ is *not-found*
+5. Otherwise, $EF$ is considered *not-found*.
+
+#### Create Collection Helper
+
+Drivers MUST support a BSON document option named `encryptedFields` for any
+[create](https://www.mongodb.com/docs/manual/reference/command/create) command helpers (e.g.
+`Database.createCollection()`). This option will be interpreted by the helper method and MUST be passed to the
+[create](https://www.mongodb.com/docs/manual/reference/command/create) command.
+
+> \[!NOTE\]
+>
+> Users are not expected to set the `escCollection` and `ecocCollection` in `encryptedFields`. SERVER-74069 added
+> server-side validation for those fields and no longer allows names to deviate from the following:
+>
+> - `enxcol_.<collectionName>.esc`
+> - `enxcol_.<collectionName>.ecoc`
+>
+> Drivers MUST NOT document the `escCollection` and `ecocCollection` options.
+
+For a helper function, `CreateCollection(collectionName, collectionOptions)` with the name of the database associated as
+`$dbName$`, look up the encrypted fields `encryptedFields` for the collection as
```

Comment:
The dollar signs were leftover from the MathML formatting. I don't think any terms should be wrapped in dollar signs within this document. Same goes for the longer GetEncryptedFields block on the next line and later sections such as "Create Encrypted Collection Helper".

If you search for this, it'd be more reliable to look for the trailing dollar sign (i.e. "$`"), since some terms legitimately start with a dollar sign.